docs/developer-guide/index.md

```diff
@@ -0,0 +1,19 @@
+---
+myst:
+  html_meta:
+    "description": "Plone developer guide"
+    "property=og:description": "Plone developer guide"
+    "property=og:title": "Developer guide"
+    "keywords": "Plone 6, developer guide, development"
+---
+
+# Developer guide
+
+This part of the documentation provides information for how to develop in Plone.
```

Comment:
````suggestion
This part of the documentation provides information for how to develop in Plone.

```{note}
Under construction. Currently documentation for developers is found throughout the documentation. We're working on moving it here.
```
````